app/models/manageiq/providers/amazon/cloud_manager/event_parser.rb

```diff
@@ -1,48 +1,45 @@
 module ManageIQ::Providers::Amazon::CloudManager::EventParser
-  # these have their own EmsEvent classes in automate
-  INSTANCE_EVENTS = %w(
-    AWS_EC2_Instance_CREATE
-    AWS_EC2_Instance_UPDATE
-    AWS_EC2_Instance_DELETE
-    AWS_EC2_Instance_running
-    AWS_EC2_Instance_shutting-down
-    AWS_EC2_Instance_stopped
-  ).to_set.freeze
-
-  def self.map_event_type(aws_event_type)
-    if INSTANCE_EVENTS.include?(aws_event_type)
-      aws_event_type
-    else
-      'ConfigurationItemChangeNotification'
-    end
+  def self.parse_config_event!(event, event_hash)
+    event_hash[:message]                   = event["configurationItemDiff"]
```

Comment:
So in all providers, we store all events that just pass the evenType check. Then you are able to change settings/automate in production, so it shows and does anything special. I think we need to keep this behavior.

In settings.yml, we can define just few interesting events. Settings is used to determine which event should show in which section in timelines. So given to requirement of 'section', we cannot do this automatically. But at any time, you can extend this in customer environment to show events of interest (they are stored just not shown in timelines)